pom.xml

```diff
@@ -924,49 +923,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.apache.rat</groupId>
-                <artifactId>apache-rat-plugin</artifactId>
-                <version>${apache.rat.version}</version>
-                <configuration>
-                    <excludeSubProjects>false</excludeSubProjects>
-                    <addDefaultLicenseMatchers>false</addDefaultLicenseMatchers>
-                    <licenses>
-                        <license implementation="org.apache.rat.analysis.license.SimplePatternBasedLicense">
-                            <licenseFamilyCategory>AL20</licenseFamilyCategory>
-                            <licenseFamilyName>Apache License, 2.0</licenseFamilyName>
-                            <patterns>
-                                <pattern>Licensed to the Apache Software Foundation (ASF)</pattern>
-                            </patterns>
-                        </license>
-                    </licenses>
-                    <licenseFamilies>
-                        <licenseFamily implementation="org.apache.rat.license.SimpleLicenseFamily">
-                            <familyName>Apache License, 2.0</familyName>
-                        </licenseFamily>
-                    </licenseFamilies>
-                    <excludes>
-                        <exclude>**/node_modules/**</exclude>
-                        <exclude>**/node/**</exclude>
-                        <exclude>**/dist/**</exclude>
-                        <exclude>**/licenses/**</exclude>
-                        <exclude>.github/**</exclude>
-                        <exclude>**/sql/soft_version</exclude>
-                        <exclude>**/common/utils/ScriptRunner.java</exclude>
-                        <exclude>**/*.json</exclude>
-                        <!-- document files -->
-                        <exclude>**/*.md</exclude>
-                        <excldue>**/*.MD</excldue>
-                        <exclude>**/*.txt</exclude>
-                        <exclude>**/docs/**</exclude>
-                        <exclude>**/*.babelrc</exclude>
-                        <exclude>**/*.eslint*</exclude>
-                        <exclude>**/.mvn/jvm.config</exclude>
-                        <exclude>**/.mvn/wrapper/**</exclude>
-                    </excludes>
-                    <consoleOutput>true</consoleOutput>
-                </configuration>
-            </plugin>
```

Comment:
Would it be better to keep the rat plugin? It can help us to conduct self-checks more conveniently in the development environment

Reply:
Self checking is convenient with license-eye via

```shell
docker run -it --rm -v $(pwd):/github/workspace apache/skywalking-eyes header check
```

and if you have files missing license headers, you can use

```
docker run -it --rm -v $(pwd):/github/workspace apache/skywalking-eyes header fix
```

to add the headers automatically for you, which RAT plugin can't.

If you don't like using docker, you can still [download the binary](https://dist.apache.org/repos/dist/release/skywalking/eyes/0.1.0/skywalking-license-eye-0.1.0-bin.tgz) and run `license-eye header check/fix` to perform the checks/fixes.

However, if you want me to keep RAT plugin, I'm OK to do so, though I don't think it's necessary to keep it as license-eye has covered what RAT has

Reply:
Deeply thanks for your answers. You are right